### PR TITLE
Gravity Forms Professional: Initial Version (Certified Connector)

### PR DESCRIPTION
**Pull request by nhance**
#   Gravity Forms Professional
_The Professional/Enterprise version of the Gravity Forms connector_

This connector uses a new API that has been completely re-built for Power Automate. This version focuses on increased security and bi-directional workflows for one of the world's most popular form platforms. This uses a fully separate API from the standard Gravity Forms connector and is intended for power users and enterprises.

**Benefits**

1. *Direct communication to end user systems*: Gravity Forms Professional connects directly to your WordPress system without ever passing form metadata or form submissions through any 3rd party servers. Your data is directly connected to Power Automate.
2. *Increased Security*: The WordPress plugin generates a secret key that is necessary to use the API. The secret key increases security by introducing a secret known only to your Power Automate connection and WordPress.
3. *Read/Write Support*: With Gravity Forms Professional, you can read/edit/delete previously submitted form entries. You can also validate and create new form entries.
4. *Note Support*: Create or fetch notes from Gravity Forms Entries.
5. *Attachment management*: Fetch Attachment content, or delete attachments after processing.
6. *Support for GravityView*: GravityView is a set of plugins that allow displaying submitted form data to site visitors. This plugin is fully supported.
7. *Update dropdown options*: You can update choices for all fields that support it using the new API.
8. *Dynamic client side control*: The connector uses dynamic schemas wherever possible, allowing the possibility for modifications on the WordPress side that extend the connector in new ways.

**Comments**
_We are the author of the Gravity Forms by reenhanced connector currently published. This connector is a more advanced version of that connector, but we have many users of the standard connector. We can commit to supporting both connectors as they use separate APIs._

---

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

**Comment by vmanoharas** [comment]
Hello @nhance, 

This PR is still work in progress and the status is showing as Draft. Please refer this link and follow the process if you would like to submit Verified Publisher connector. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-5-submit-your-connector-for-deployment

**Comment by nhance** [comment]
@vmanoharas This connector is now ready for review.

File: certified-connectors/Gravity Forms Professional/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1657 @@
...
+          }
+        }
+      }
+    },
+    "/resources/forms": {
+      "get": {
+        "operationId": "GetForms",
+        "tags": ["Forms"],
+        "summary": "Get forms",
+        "description": "Get a list of Gravity Forms forms",
```
Comment (vmanoharas) [inline certified-connectors/Gravity Forms Professional/apiDefinition.swagger.json]: Is this repeated word? if so, can you please fix it?

**Review by vmanoharas — COMMENTED** [review]
Hello @nhance,

Can you please resolve review comments?

File: certified-connectors/Gravity Forms Professional/apiDefinition.swagger.json (hunk 1 above)
Comment (nhance) [reply certified-connectors/Gravity Forms Professional/apiDefinition.swagger.json]: Hi, the product name is "Gravity Forms", and it has multiple different entities under it, the "form" being one of the entities.

This is correct, but if you like we can just make this read "Get a list of forms". Will update.

**Comment by vmanoharas** [comment]
Hello @nhance, 

I haven't seen your registration of this connector, as per the Step 1 in this link.  If you would like to publish your connector as Certified connector could you please follow the process?

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-1-register-your-connector

**Comment by nhance** [comment]
> Hello @nhance,
> 
> I haven't seen your registration of this connector, as per the Step 1 in this link. If you would like to publish your connector as Certified connector could you please follow the process?

Hi @vmanoharas,

My apologies! We have submitted the form and are ready for certification. Thank you

**Review by vmanoharas — APPROVED** [review]
Hello @nhance 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**